middle_end/flambda2/to_jsir/to_jsir_static_const.ml

```diff
@@ -0,0 +1,90 @@
+let static_const_not_supported () =
+  Misc.fatal_error "This static_const is not yet supported."
+
+let block_like ~env ~res (const : Flambda2_terms.Static_const.t) :
+    Jsir.expr * To_jsir_env.t * To_jsir_result.t =
+  match const with
+  | Set_of_closures _closures ->
+    Misc.fatal_errorf
+      "Cannot translate %a: expected a block-like static const, instead found \
+       Set_of_closures"
+      Flambda2_terms.Static_const.print const
+  | Block (tag, mut, _shape, fields) ->
+    (* CR selee: is it ok to ignore shape? *)
+    let tag = Flambda2_kinds.Tag.Scannable.to_int tag in
+    let mutability =
```

Comment:
Here we tend to write `let mutability : Jsir.mutability =` as then you can skip the `Jsir.` qualifications on the branches.